homeassistant/components/overkiz/cover_entities/vertical_cover.py

```diff
@@ -36,6 +37,19 @@
 class VerticalCover(OverkizGenericCover):
```

Comment:
If you made this the base class and created `LowSpeedVerticalCover` you could avoid some of the branching in the `async def async` functions

Reply:
Indeed, it ease the code. Done with https://github.com/home-assistant/core/pull/66750/commits/b41ff6288730ec1f4e2e2b106a113dbc8694a85e